docs/sources/developers/plugins/add-distributed-tracing-for-backend-plugins.md

```diff
@@ -4,42 +4,41 @@ title: Add distributed tracing for backend plugins
-and with the same propagation format as the main Grafana instance. The tracing context is also propagated from the Grafana instance
-to the plugin, so the plugin's spans will be correlated to the correct trace.
+## Plugin configuration
 
-## Configuration
-
-> **Note:** Only OpenTelemetry is supported. If Grafana is configured to use a deprecated tracing system (Jaeger or OpenTracing),
-> tracing will be disabled in the plugin. Please note that OpenTelemetry + Jaeger propagator is supported.
-
-OpenTelemetry must be enabled and configured for the Grafana instance. Please refer to [this section](
-{{< relref "../../setup-grafana/configure-grafana/#tracingopentelemetry" >}}) for more information.
-
-As of Grafana 9.5.0, plugins tracing must be enabled manually on a per-plugin basis, by specifying `tracing = true` in the plugin's config section:
+Plugin tracing must be enabled manually on a per-plugin basis, by specifying `tracing = true` in the plugin's config section:
 
 ```ini
 [plugin.myorg-myplugin-datasource]
 tracing = true
 ```
 
-## Implementing tracing in your plugin
+> **Note:** If tracing is disabled in Grafana, `backend.DefaultTracer()` returns a no-op tracer.
+
+### OpenTelemetry configuration
+
+Grafana supports [OpenTelemetry](https://opentelemetry.io/) for distributed tracing. If Grafana is configured to use a deprecated tracing system (Jaeger or OpenTracing), then tracing is disabled in the plugin. 
+
+> **Note:** Although Grafana doesn't support Jaeger, it supports [OpenTelemetry Jaeger propagator](https://www.npmjs.com/package/@opentelemetry/propagator-jaeger) for HTTP header propagation.
+
+OpenTelemetry must be enabled and configured for the Grafana instance. Please refer to the [Grafana configuration documentation](
+{{< relref "../../setup-grafana/configure-grafana/#tracingopentelemetry" >}}) for more information.
+
+Refer to the [OpenTelemetry Go SDK](https://pkg.go.dev/go.opentelemetry.io/otel) for in-depth documentation about all the features provided by OpenTelemetry.
+
+## Implement tracing in your plugin
 
 > **Note:** Make sure you are using at least grafana-plugin-sdk-go v0.157.0. You can update with `go get -u github.com/grafana/grafana-plugin-sdk-go`.
 
-When OpenTelemetry tracing is enabled on the main Grafana instance and tracing is enabled for a plugin,
-the OpenTelemetry endpoint address and propagation format will be passed to the plugin during startup,
-which will be used to configure a global tracer.
+### Configure a global tracer
 
-1. The global tracer is configured automatically if you use <code>datasource.Manage</code> or <code>app.Manage</code> to run your plugin.
+When OpenTelemetry tracing is enabled on the main Grafana instance and tracing is enabled for a plugin, the OpenTelemetry endpoint address and propagation format is passed to the plugin during startup. These parameters are used to configure a global tracer.
```

Comment:
> These parameters are used to configure a global tracer.

Yes this is technically true. The global tracer is provided by the SDK and configured when calling `datasource.Manage` | `app.Manage`. There _is_ another way to enable tracing without using `datasource/app.Manage`, but we want to discourage that because those plugins won't use our instance management and that comes with other limitations.

For this reason we originally decided to remove from this doc page all references to "configuring tracing manually", or using other tracers that are not provided by the SDK, and only provided examples using `datasource/app.Manage`